### ISSUE TITLE
why the training scripts do not set the fix_attn_mask to True?

### ISSUE THREAD

**Issue by BIGKnight** [zhenyi4/codi#4]
I carefully looked through the code, and I found that during the training stage, the attention mask of the teacher task (ref output) is None since you set the fix_attn_mask as false by default, which is a little bit strange to me.

**Comment by zhenyi4** [comment]
Thanks for your question.

I set it to False because it didn’t affect performance in my experiments, and this setting is consistent with my previous experimental configurations.
